crates/ide_assists/src/handlers/move_from_mod_rs.rs

```diff
@@ -0,0 +1,131 @@
+use ide_db::{
+    assists::{AssistId, AssistKind},
+    base_db::AnchoredPathBuf,
+};
+use syntax::{ast, AstNode, TextRange};
+
+use crate::assist_context::{AssistContext, Assists};
+use crate::utils::trimmed_text_range;
```

Comment:
Nit:
```suggestion
use crate::{assist_context::{AssistContext, Assists}, utils::trimmed_text_range};
```
We use deep nested import style